R/qza_to_tse.R

```diff
@@ -0,0 +1,107 @@
+#' Generates a TreeSummarizedExperiment (TSE) object from .qza artifacts
+#'\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'
+#' Construct a TreeSummarizedExperiment object from multiple 
```

Comment:
add linking to TreeSummarizeExperiment here

Reply:
done